### PR TITLE
Added Time Complexity and remove single comment into multi comment

### PR DESCRIPTION
**Pull request by UG-SEP**
hey,
I have done some changes in Permutation of string.c file please review it

### CODE REVIEW

File: strings/c-or-cpp/Permutation-of-String.c [hunk 1]
```diff
@@ -55,11 +55,13 @@ void swap(char *a,char *b)
...
-//        6(cab)
+/* Example:
+  Input: abc
+  Output: 1(abc)
+          2(acb)
+          3(bac)
+          4(bca)
+          5(cba)
+          6(cab)
+Time Complexity: O(n*n!)
```
Comment (UG-SEP) [inline strings/c-or-cpp/Permutation-of-String.c:66]: is this time complexity is correct

File: strings/c-or-cpp/Permutation-of-String.c [hunk 2]
```diff
@@ -55,11 +55,13 @@ void swap(char *a,char *b)
...
 }
-// Example:
-//Input: abc
-//Output: 1(abc)
-//        2(acb)
-//        3(bac)
-//        4(bca)
-//        5(cba)
-//        6(cab)
+/* Example:
```
Comment (UG-SEP) [inline strings/c-or-cpp/Permutation-of-String.c:58]: did the multi comment looks good or not

File: strings/c-or-cpp/Permutation-of-String.c [hunk 3]
```diff
@@ -55,11 +55,13 @@ void swap(char *a,char *b)
...
+  Input: abc
+  Output: 1(abc)
+          2(acb)
+          3(bac)
+          4(bca)
+          5(cba)
+          6(cab)
+Time Complexity: O(n*n!)
+*/
```
Comment (UG-SEP) [inline strings/c-or-cpp/Permutation-of-String.c:67]: did i need to add new line at the end

File: strings/c-or-cpp/Permutation-of-String.c (hunk 2 above)
Comment (Arsenic-ATG) [reply strings/c-or-cpp/Permutation-of-String.c:58]: yes, multiline comments looks visually better.

File: strings/c-or-cpp/Permutation-of-String.c (hunk 1 above)
Comment (Arsenic-ATG) [reply strings/c-or-cpp/Permutation-of-String.c:66]: looks correct to me

File: strings/c-or-cpp/Permutation-of-String.c (hunk 3 above)
Comment (Arsenic-ATG) [reply strings/c-or-cpp/Permutation-of-String.c:67]: Although it is recommended to end the file with newline character, It is no hard and fast rule. 

The reason behind this lies in the POSSIX definition of a line where by convention, termination of line is defined by a newline (or more formally end of line <EOL> ) character, so your last line would not be considered as a line till the time you don't add newline at the end of it.

Although this should not be a problem till the time you are not using unix tools ( like concatenation, which might give undesired results when used with files with no newline at the end) , It is recommended to follow the convention and add a newline at the end of the file.

**Review by Arsenic-ATG — DISMISSED** [review]

**Comment by UG-SEP** [comment]
@Arsenic-ATG 
> @UG-SEP UG-SEP dismissed Arsenic-ATG’s stale review via db9b385 10 minutes ago
what happen i haven't done anything just added new line but how It dismissed you stale review via db9b385

**Comment by Arsenic-ATG** [comment]
> what happen i haven't done anything just added new line but how It dismissed you stale review via db9b385

yes, it't not your fault

It is GitHub's ~~bug~~ feature where it dismisses all the code review on the branch whenever a new commit is added to it ( you can see this for more info (https://github.com/isaacs/github/issues/1157) )

not worries. I will approve it again.

**Review by Arsenic-ATG — APPROVED** [review]

**Comment by UG-SEP** [comment]
what a bug😁😁😂

**Comment by ming-tsai** [comment]
Great jobs @UG-SEP

**Pull request merged**